src/Orleans.Membership.Azure.TableStorage/AzureBasedMembershipTable.cs

```diff
@@ -9,9 +9,9 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Orleans.AzureUtils.Configuration;
-using Orleans.Configuration;
 using LogLevel = Microsoft.Extensions.Logging.LogLevel;
 using Orleans.Runtime.Configuration;
+using Orleans.Membership.Azure.TableStorage;
```

Comment:
perhaps also define `using ErrorCode = Orleans.Membership.Azure.TableStorage.Utulities.ErrorCode; ` to avoid such verbosity thought out the file (and undo all changes here, other than the namespace)?

Reply:
That doesn't work because of the common Orleans namespace and it is ambiguous, but to reduce verbosity I'm adding a ```TableStorageErrorCode``` alias.